src/App.tsx

```diff
@@ -1,25 +1,24 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { TodoListItem } from "./TodoListItem";
+
+import { Todo } from './types';
+
+const todos: Todo[] = [
+  {
+    text: 'Wynieść śmieci',
+    complete: true
+  },
+  {
+    text: 'Nauczyć się Reacta by Kamil',
+    complete: false
+  }
+];
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+  return(
+      <ul>
+          {todos.map((data) => <TodoListItem todo={data} />)}
```

Comment:
keys are quite important part of making lists. You may not see the difference now, but it may bite you near feature :P https://beta.reactjs.org/learn/rendering-lists#keeping-list-items-in-order-with-key